griptape/common/__init__.py

```diff
@@ -18,6 +18,7 @@
 
 from .reference import Reference
 
+from .observable import observable, Observable
```

Comment:
Should these be moved to the `observability` package?

Reply:
I believe this choice was to mitigate a circular reference between the `Observability` context manager and the Observability Drivers. I can double check if you want